packages/internal/test/test-storage-test-utils.js

```diff
@@ -262,6 +281,19 @@ test('makeFakeStorageKit sequence data', async t => {
     [{ method: 'append', args: [[deepPath, 'qux']] }],
     'manual-sequence grandchild setValue message',
   );
+
+  await childNode.delete();
+  t.deepEqual(
+    messages.slice(-1),
+    [{ method: 'delete', args: [childPath] }],
```

Comment:
this seems to verify that a message crosses a bridge.

How about a a test where we see something like 3 child nodes a, b, and c; then we delete b and we see only a and c?

Reply:
This test runs against a fake backend, so such a test wouldn't be indicative of much (and also, the StorageNode interface does not provide a means of enumerating children). But verification of effects upon chain storage data when messages such as this are received is included in the changes to golang/cosmos/x/vstorage/vstorage_test.go.